code/game/objects/items/toys.dm

```diff
@@ -134,6 +134,31 @@
 	force = 0
 	var/random_color = TRUE
 
+/obj/item/toy/balloon/attackby(obj/item/I, mob/user, params)
+	if(istype(I, /obj/item/ammo_casing/caseless/foam_dart) && ismonkey(user))
+		pop_balloon(monkey_pop = TRUE)
+	else
+		return ..()
+
+/obj/item/toy/balloon/hitby(atom/movable/AM, skipcatch, hitpush, blocked, datum/thrownthing/throwingdatum)
+	if(ismonkey(throwingdatum.thrower) && istype(AM, /obj/item/ammo_casing/caseless/foam_dart))
+		pop_balloon(monkey_pop = TRUE)
+	else
+		return ..()
+
+/obj/item/toy/balloon/bullet_act(obj/projectile/P)
+	if((istype(P,/obj/projectile/bullet/p50) || istype(P,/obj/projectile/bullet/reusable/foam_dart)) && ismonkey(P.firer))
```

Comment:
why only p50s instead of all bullets?

Reply:
Sniper monkeys.

When was the last time you saw a monkey with a Glock in Bloons

Reply:
In bloons td battles one, The cobra agent was a monkey with a glock

Reply:
and there's, like, a gatling gun monkey

Reply:
Is there a gatling gun in SS13??